src/AutoEncoderModel.cpp

```diff
@@ -124,6 +124,15 @@ AutoEncoderModel::AutoEncoderModel() :
     else {
         printf("No %s found. Initializing new flow decoder model.\n", flowDecoderFilename); // TODO logging
     }
+
+    // Create output directories if they do not exist
+    const std::string modelsRoot{"models"};
```

Comment:
Maybe this should be made into a `constexpr` variable into the `Constants.hpp` along with all the other constants?